pkg/proxy/winkernel/proxier.go

```diff
@@ -152,11 +152,13 @@ const (
 	MAX_COUNT_STALE_LOADBALANCERS = 20
 )
```

Comment:
There are 2 methods in proxier.go that I believe are still calling hcsshim 
  * deleteAllHnsLoadBalancerPolicy() 
  * getHnsNetworkInfo() 
   
  Can we update these to also use HcnService interface methods?

Reply:
Addressed. Func: getHnsNetworkInfo() is removed as it is never used